### PR TITLE
Backend-Hotfix: Swapping "local" URI with environment variable DATABASE_URL

### PR DESCRIPTION
**Pull request by albertocruz6**
***HOTFIX***
- The current deployed branch has a "config" folder with the postgresql URI from the Heroku app. Heroku changes the credentials randomly every 1 or 2 weeks, forcing us to continuously update our URI manually which is inefficient. Luckily for us Heroku stores the updated URI in a environmental variable called DATABASE_URL that can be used to fetch it on runtime. This hotfix adds this functionality. 
    - In the teams private channels I'll send a Makefile to run the program locally while managing to fetch the remote variables from the server in order to not rely on the config folder
- Small change to .gitignore

### CODE REVIEW

**Comment by fabiolarobles1** [comment]
Before merging, was the connection tested and is still working properly?

File: backend/model/users.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
+import os
 import psycopg2
 from config.dbconfig import pg_config
```
Comment (fabiolarobles1) [inline backend/model/users.py]: nit: delete import if no longer used

Comment (albertocruz6) [reply backend/model/users.py]: Done

**Comment by albertocruz6** [comment]
> Before merging, was the connection tested and is still working properly?

It need the environment variable to make the call succesfully, but it works

**Pull request merged**